lib/src/button/button.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'colors';
+@import '../sass/cashmere';
```

Comment:
if you include the whole cashmere bundle in components you duplicate all the styles. I would suggest to only include what you need, ie `functions`

also with `includePaths` you no longer have to remember the relative path to the common sass files, you can just do `@import 'functions';`

At least it appears that way when I debug
![image](https://user-images.githubusercontent.com/1934453/36317233-ceba9aae-12f9-11e8-8b02-18538e1e2bc3.png)